bin/spice/cmd/chat.go

```diff
@@ -225,17 +225,36 @@ spice chat --model <model> "What is Spice.ai?"
 			cmd.Printf("Using model: %s\n", selectedModel)
 			model = selectedModel
 		} else {
-			modelIsAvailable := false
-			for _, availableModel := range availableModels {
-				if availableModel == model {
-					modelIsAvailable = true
+			is_configured := false
+			for _, m := range models.Data {
+				if m.Id == model {
+					is_configured = true
 					break
 				}
 			}
 
-			if !modelIsAvailable {
-				slog.Error("Requested model is not available", "model", model)
-				slog.Info("Available models", "models", strings.Join(availableModels, ", "))
+			if !is_configured {
+				ids := make([]string, len(models.Data))
+				for i, m := range models.Data {
+					ids[i] = m.Id
+				}
+
+				slog.Error(fmt.Sprintf("model %s does not exist — configured models: %s",
+					model, strings.Join(ids, ", ")))
+
+				os.Exit(1)
+			}
+
+			is_available := false
```

Comment:
Same comment.

Reply:
Use a consistent name. In this case it's testing for readiness, so use "ready" not "available" which can mean something else.